common-app-lib/src/cmr/common_app/cache/consistent_cache.clj

```diff
@@ -144,12 +144,9 @@
    (create-consistent-cache nil))
   ([options]
    (let [timeout (get options :hash-timeout-seconds (consistent-cache-default-hash-timeout-seconds))
-         hash-cache (fallback-with-timeout (cubby-cache/create-cubby-cache) timeout)
+         hash-cache (cubby-cache/create-cubby-cache options)
          main-cache (mem-cache/create-in-memory-cache)]
-     (create-consistent-cache main-cache hash-cache)))
+     (create-consistent-cache main-cache (fallback-with-timeout hash-cache timeout))))
```

Comment:
Nice little bit of cleaning; we should do more of this :-) Makes it easier to read.